integration/setup/config.go

```diff
@@ -144,7 +143,7 @@ func NewConfig() (Config, error) {
 			HelmClient: helmClient,
 			Logger:     logger,
 
-			Namespace: namespace,
+			Namespace: env.TargetNamespace(),
```

Comment:
FMI, what is the resource package actually good for and doing? I do not see it being used at all.

Reply:
It's for transition period. It should be deleted when it's not in use.

Reply:
What transition? I do not see where it is used at all.

Reply:
Can be removed now. I added it and wanted to use it for `helmclient` before it got renamed to `release`.

Then the whole `clusterRoleBinding` trouble happened. IMO we can just remove it in a follow up PR.